Issue where it crashes, citing a random txt file

```
[spiderunderurbed@daspidercave:~]$ tuido
panic: open /home/spiderunderurbed/tmp/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/SteamLinuxRuntime_sniper/var/tmp-MR4NT2/usr/share/X11/rgb.txt: no such file or directory

goroutine 1 [running]:
github.com/nilock/tuido/tui.getItems({0xc000167860, 0x9d})
        github.com/nilock/tuido/tui/tui.go:458 +0x2d3
github.com/nilock/tuido/tui.Run()
        github.com/nilock/tuido/tui/tui.go:65 +0x39f
main.main()
        github.com/nilock/tuido/main.go:22 +0x99
[ble: exit 2]

[spiderunderurbed@daspidercave:~]$ cd /etc/nixos

[spiderunderurbed@daspidercave:/etc/nixos]$ tuido

[spiderunderurbed@daspidercave:/etc/nixos]$ 
```
Tuido for some reason crashed when I ran it in my home directory, but if I run it inside my root directory I do not get the error above.

--- a/tui/tui.go
+++ b/tui/tui.go
@@ -451,24 +451,24 @@ func (t tui) Init() tea.Cmd { return tick() }
 func getItems(file string) []*tuido.Item {
 	items := []*tuido.Item{}
 
-	f, err := os.Open(file)
-	defer f.Close()
+	if f, err := os.Open(file); err != nil {
+		f.Close()
+		return items
+	} else {
+		defer f.Close()
 
-	if err != nil {
-		panic(err)
-	}
-
-	scanner := bufio.NewScanner(f)
-	line := 1
-	for scanner.Scan() {
-		if tuido.IsTuido(scanner.Text()) {
-			item := tuido.New(file, line, scanner.Text())
-			items = append(items, &item)
+		scanner := bufio.NewScanner(f)
+		line := 1
+		for scanner.Scan() {
+			if tuido.IsTuido(scanner.Text()) {
+				item := tuido.New(file, line, scanner.Text())
+				items = append(items, &item)
+			}
+			line++
 		}
-		line++
-	}
 
-	return items
+		return items
+	}
 }
 
 func getFiles(wd string, extensions []string) []string {

